### ISSUE TITLE
sjk bottle files missing

### ISSUE THREAD

**Issue by gaoyifan** [Homebrew/homebrew-core#13281]
The following bottle files are missing:

```
http://homebrew.bintray.com/bottles/sjk-0.5.1.sierra.bottle.tar.gz
http://homebrew.bintray.com/bottles/sjk-0.5.1.el_capitan.bottle.tar.gz
http://homebrew.bintray.com/bottles/sjk-0.5.1.yosemite.bottle.tar.gz
```

return 404 when installing...

maybe we need to trigger agi

# Please always follow these steps:
- [X] Confirmed this is a problem with only one, specific formula and not Homebrew/brew? If it's a general Homebrew/brew problem please file this issue at https://github.com/Homebrew/brew/issues/new
- [X] Ran `brew update` and retried your prior step?
- [X] Ran `brew doctor`, fixed as many issues as possible and retried your prior step?
- [ ] If `brew gist-logs` didn't work: ran `brew config` and `brew doctor` and included their output with your issue? no helpness for the problem.

To help us debug your issue please explain:
- What you were trying to do (and why)
- What happened (include command output)
- What you expected to happen
- Step-by-step reproduction instructions (by running `brew install` commands)

**Please note we may immediately close your issue without comment if you do not fill out the issue template and provide ALL the requested information.**

**Comment by fxcoudert** [comment]
#11264 was merged without its bottles

**Comment by fxcoudert** [comment]
Will be fixed by #13287
